setupext/utils.py

```diff
@@ -18,17 +18,17 @@
 # *****************************************************************************
 import os
 import codecs
+import glob
 
 
 def read_utf8(path, *parts):
     filename = os.path.join(os.path.dirname(path), *parts)
     return codecs.open(filename, encoding='utf-8').read()
 
 
-def find_sources():
+def find_sources(roots=[]):
```

Comment:
Awooga! I just bisected an issue (to be raised shortly) which points back to this commit. I have no idea if the problem is related to this, but I was quickly reviewing the code and saw this. I'm sure you already know about the pitfalls of mutable defaults with Python, and that they are a no-go unless you **really** intend them.

Reply:
The other mutable defaults in Platform appear to have been removed in 1f58b10667b55f01af19327d5ce6d155c2edfe4f.

Issue mentioned opened in #848.  Unfortunately, fixing this one doesn't seem to address